packages/apollo-gateway/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 - Drop use of `loglevel-debug`.  This removes the very long date and time prefix in front of each log line and also the support for the `DEBUG=apollo-gateway:` environment variable.  Both of these were uncommonly necessary or seldom used (with the environment variable also being undocumented).  The existing behavior can be preserved by providing a `logger` that uses `loglevel-debug`, if desired, and more details can be found in the PR.  [PR #3896](https://github.com/apollographql/apollo-server/pull/3896)
 - Fix Typescript generic typing for datasource contexts [#3865](https://github.com/apollographql/apollo-server/pull/3865) This is a fix for the `TContext` typings of the gateway's exposed `GraphQLDataSource` implementations. In their current form, they don't work as intended, or in any manner that's useful for typing the `context` property throughout the class methods. This introduces a type argument `TContext` to the class itself (which defaults to `Record<string, any>` for existing implementations) and removes the non-operational type arguments on the class methods themselves.
 - Implement retry logic for requests to GCS [PR #3836](https://github.com/apollographql/apollo-server/pull/3836) Note: coupled with this change is a small alteration in how the gateway polls GCS for updates in managed mode. Previously, the tick was on a specific interval. Now, every tick starts after the round of fetches to GCS completes. For more details, see the linked PR.
+- Gateway issues health checks to downstream services via `performServiceHealthCheck` configuration option [#3930](https://github.com/apollographql/apollo-server/pull/3930) Note: expected behavior differs between managed and unmanaged federation. See PR for new test cases and documentation.
```

Comment:
I feel like we slowly started breaking our long-standing pattern of putting `[#XXXX](link-to-pr)` at the end of each changelog line and have started interweaving it.   Just noting that, but curious if there's a reason.

I might suggest:

```suggestion
- Introduce `performServiceHealthCheck` option which validates the readiness of downstream services prior to placing a schema in service.  The precise behavior differs between managed and unmanaged federation, and additional details are available in the related documentation and PR. [#3930](https://github.com/apollographql/apollo-server/pull/3930)
```

(... though I think before we finalize this release we should probably introduce some "Breaking" and "New" designators/headings, but that's beyond the scope of this PR.)

Reply:
Totally. I may have started that trend, considering everything after the link "details". In thinking about it, I don't think that's helpful. I'd rather see the link consistently at the end, and chances are if there are "details", they're probably not just an afterthought!

I couldn't commit your suggestion, for some reason it became outdated. 3e32db8